docroot/sites/all/modules/features/ilr_forms_students/ilr_forms_students.features.field_instance.inc

```diff
@@ -5599,7 +5599,7 @@ function ilr_forms_students_field_default_field_instances() {
           'field_formatter_label' => 'WISP street address line 1',
         ),
         'type' => 'text_default',
-        'weight' => 48,
+        'weight' => 49,
```

Comment:
Too bad about all these weight value changes. Makes for a very noisy diff and almost made me ignore this file, where there are some real changes that I think we need to look at.